web/src/components/LandingPageTiles.test.tsx

```diff
@@ -14,26 +14,39 @@ describe("<LandingPageTiles />", () => {
   });
 
   it("routes user to industry selection when the starting a business button is clicked", async () => {
-    render(<LandingPageTiles />);
+    render(<LandingPageTiles isWelcomePage={false} />);
 
     fireEvent.click(screen.getByText(Config.landingPage.landingPageTile2Text));
 
     expect(mockPush).toHaveBeenCalledWith({ pathname: "/onboarding", query: { flow: "starting" } });
   });
 
   it("routes user to out-of-state business section when the out-of-state business button is clicked", async () => {
-    render(<LandingPageTiles />);
+    render(<LandingPageTiles isWelcomePage={false} />);
 
     fireEvent.click(screen.getByText(Config.landingPage.landingPageTile4Text));
 
     expect(mockPush).toHaveBeenCalledWith({ pathname: "/onboarding", query: { flow: "out-of-state" } });
   });
 
   it("routes user to business status section when the running a business button is clicked", async () => {
-    render(<LandingPageTiles />);
+    render(<LandingPageTiles isWelcomePage={false} />);
 
     fireEvent.click(screen.getByText(Config.landingPage.landingPageTile3Text));
 
     expect(mockPush).toHaveBeenCalledWith({ pathname: "/onboarding", query: { flow: "up-and-running" } });
   });
+
+  it("shows only 'Get Started' and 'Pay Taxes' tiles when visiting welcome page URL", () => {
+    useMockRouter({ pathname: "/welcome" });
```

Comment:
Given that the logic that controls the welcome page is determined by `isWelcomePage`, I don't think you need `useMockRouter`.